openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -343,6 +343,75 @@ Note: The issuer MAY decide to support both options. In which case, it is at the
 
 A Credential Format Profile for Credentials complying with IETF SD-JWT VCs [@!I-D.ietf-oauth-sd-jwt-vc] is defined in Annex A.3 of [@!OIDF.OID4VCI] and Annex A.4 of [@!OIDF.OID4VP].
 
+## SD-JWT VC Data Model (SD-JWT VCDM)
+
+SD-JWT VCDM is a data model that follows IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc], but allows the usage of [@!W3C.VCDM1.1] or [@!W3C.VCDM2.0]. When IETF SD-JWT VC is mentioned in this specification, SD-JWT VCDM define in this section MAY be used.
```

Comment:
need to add more detailed explanation that sd-jwt processor is used to process sd-jwt to get the payload, and json-ld processor is only applied to the vcdm part of the payload. also clarify that sd-jwt before processing (the one that includes _sd etc) should not be put into a json-ld processor.

Reply:
@awoie to propose a spcific text on processing :)

Reply:
Done, see here https://github.com/openid/oid4vc-haip/pull/147#discussion_r1922554997

Reply:
thank you!

Reply:
Proposal to resolve the problem
https://github.com/openid/oid4vc-haip/pull/147#discussion_r1932025108

Reply:
ok, let's continue the discussion there